### ISSUE TITLE
Provide a Button to refetch the device list

### ISSUE THREAD

**Issue by Ghabry** [element-hq/element-web#14272]
**Is your suggestion related to a problem? Please describe.**
See: https://github.com/vector-im/riot-web/issues/13824#issuecomment-651824685

Fixing this issue requires database write access which is an advanced topic and many users won't have write access.

**Describe the solution you'd like**

Provide a button anywhere in the Riot-Web (or Desktop) Ui to Refetch the device list.

It could be hidden somewhere in the Advanced options because it is a troubleshooting feature.

**Comment by t3chguy** [comment]
I don't think this is something the client can do anything about as their server will still serve them the outdated devicelist which needs to be refreshed from the originating server.

**Comment by Ghabry** [comment]
Ah I thought this is some client feature which works by checking through some API if a key refresh is pending. Yeah then this is the wrong repo for this. Any better repository to file this issue?

So this tells instead to the server that it shall refetch the keys the next time a client asks for them?
Yeah okay then this is of course out of scope for the client...

Maybe there should be some JSON endpoint a server administrator can POST to to force a refetch? This could be exposed to the client.

**Comment by t3chguy** [comment]
https://github.com/matrix-org/matrix-doc/pull/2638
